server/src/main/java/org/elasticsearch/index/mapper/KeywordFieldMapper.java

```diff
@@ -72,23 +75,36 @@
 
         public static final String NULL_VALUE = null;
         public static final int IGNORE_ABOVE = Integer.MAX_VALUE;
+        public static final boolean EAGER_GLOBAL_ORDINALS = false;
+        public static final boolean SPLIT_QUERIES_ON_WHITESPACE = false;
+    }
+
+    public static class KeywordField extends Field {
+
+        public KeywordField(String field, BytesRef term, FieldType ft) {
+            super(field, term, ft);
+        }
+
```

Comment:
We have a few places that were doing instanceof checks on lucene IndexableField fieldtypes - this allows us to continue doing those for now, as we can instead do an instanceof check for KeywordField